charts/gcp/values.yaml

```diff
@@ -37,6 +37,19 @@ credentials:
 
 skipTLSVerify: false
 
+# @schema
+# type: object
+# additionalProperties: true
+# @schema
+defaultTransform:
+  changes:
+    exclude:
+      - change_type == 'diff' &&  summary in ['updateTime', 'settings.settingsVersion']
```

Comment:
Can we implement this as a ScrapePlugin ?

Reply:
Have updated